src/js/edu-benefits/pages/schoolSelection.js

```diff
@@ -0,0 +1,72 @@
+import _ from 'lodash/fp';
+
+import * as educationProgram from '../definitions/educationProgram';
+import { uiSchema as uiSchemaDate } from '../../common/schemaform/definitions/date';
+import { states } from '../../common/utils/options-for-select';
+
+const stateLabels = states.USA.reduce((current, { label, value }) => {
+  return _.merge(current, { [value]: label });
+}, {});
+
+export default function createSchoolSelectionPage(
+  schema,
+  fields = ['educationProgram', 'educationObjective', 'nonVaAssistance'],
```

Comment:
@raquelromano-gov Whenever the `vets-json-schema` changes, I've found this happens to me. Running `npm install` and restarting the watch task like @jbalboni says usually does the trick.